docs/Contributing/API-for-contributors.md

```diff
@@ -536,54 +536,114 @@ The MDM endpoints exist to support the related command-line interface sub-comman
 - [Match preassigned profiles](#match-preassigned-profiles)
 - [Get FileVault statistics](#get-filevault-statistics)
 
-### Generate Apple DEP Key Pair
+### Generate Apple Business Manager public key (AED)
+
+`GET /api/v1/fleet/mdm/apple/dep/public_key`
 
 #### Parameters
 
-None.
+| Name | Type    | In   | Description                             |
+| ---- | ------- | ---- | --------------------------------------- |
+| alt   | string | query | **Required.** Set to "media" to download the public key for ABM. |
 
-#### Example
+#### Example (download public key)
 
-`POST /api/v1/fleet/mdm/apple/dep/key_pair`
+`GET /api/v1/fleet/mdm/apple/dep/public_key?alt=media`
```

Comment:
@marko-lisica since we're completely changing this endpoint, would it make sense to move away from `dep`? the new name is ADE, but maybe we could use `automatic` like we do for the rest of the endpoints?

Reply:
@roberto Good point, DEP term is deprecated. I'm looking at the rest of the docs and I think we should go for `apple/abm/public_key`. This should be the best, since this is public key for ABM. What do you think?